pkg/server/k8s/client.go

```diff
@@ -0,0 +1,182 @@
+package k8s
+
+import (
+	"encoding/json"
+
+	"github.com/luizalabs/teresa-api/pkg/server/app"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/pkg/api/resource"
+	k8sv1 "k8s.io/client-go/pkg/api/v1"
+	asv1 "k8s.io/client-go/pkg/apis/autoscaling/v1"
+	restclient "k8s.io/client-go/rest"
+)
+
+type k8sClient struct {
+	kc *kubernetes.Clientset
+}
+
+func newNs(a *app.App, user string) *k8sv1.Namespace {
+	return &k8sv1.Namespace{
+		ObjectMeta: k8sv1.ObjectMeta{
+			Name: a.Name,
+			Labels: map[string]string{
+				"teresa.io/team": a.Team,
```

Comment:
Maybe I found a bug on #205 .
I was getting app team for `teresa.io/app` annotation 😕 .